leistungsbeschreibung/README.md

```diff
@@ -1,57 +1,57 @@
+Zunächst wird über die fiskaltrust.Middleware sichergestellt, dass alle Belege von einer dritten Partei (fiskaltrust) neben dem Kassenhersteller und dem Kassenhändler verarbeitet werden. Dies nennen wir die organisatorische Implementierung des Schutzes vor Manipulation.
 
+Zur technische Implementierung des Schutzes vor Manipulation wird jeder Request von dem Kassensystem und auch jeder Response von der fiskaltrust.Middleware gehasht, wodurch die Datenintegrität der Requests und Responses sichergestellt wird. Um nachweisen zu können, dass keine Manipulation stattgefunden hat, wird ein weiterer Hash-Wert erzeugt, der sich auf die gesamte Request-Response-Kette bezieht. Dieser wird aus folgenden Werten generiert:
+- der eindeutigen Identifikation der Kette
+- dem Erstellungszeitpunkt
+- der Belegnummer
+- der Hash-Werte des Request und Response 
+- sowohl als auch dem Hash-Wert des vorherigen Blocks. 
 
+Diese Verkettung bietet die Möglichkeit, Manipulationen sicher zu erkennen.
 
  ![receipt-chain](../product-service-description/compliance-as-a-service/media/receipt-chain.png)
 
- 
 
-Um das Risiko eines Angriffs auf die Kette ab dem letzten nicht verketteten Hashwert zu begrenzen, bietet fiskaltrust einen Mechanismus, der die aktuellen Daten in die fiskaltrust-Cloud spiegelt. Dieser Datenspiegel bietet die Möglichkeit, Angriffe zu erkennen, die an der Registrierkasse selbst nicht erkennbar wären.
+Um das Risiko einer Manipulation der Kette auf den letzten nicht verketteten Hash-Wert zu begrenzen, stellt fiskaltrust einen Mechanismus bereit, der die aktuellen Daten in die fiskaltrust.Cloud hochlädt und dadurch spiegelt. Diese Datenspiegelung ermöglicht es, Manipulationen zu erkennen, die an der Registrierkasse selbst nicht erkennbar wären.
 
-Als letzte Komponente des fiskaltrust-Sicherheitsmechanismus bietet die fiskaltrust-Middleware auch marktbezogene Sicherheitsmechanismen.
+Als weitere Komponente des fiskaltrust.SecurityMechanism stellt die fiskaltrust.Middleware marktbezogene Sicherheitsmechanismen zur Verfügung. In Deutschland wird zum Beispiel die TSE (Technische Sicherheitseinrichtung) über die SCU (Security Creation Unit) der fiskaltrust.Middleware angebunden.
 
-Um für verschiedene Plattformen und Betriebssysteme so offen wie möglich zu sein und das Versprechen zu erfüllen, als stabile Schnittstelle zum Possystem zu fungieren, folgt die fiskaltrust-Middleware einer strengen Architektur.
-
- 
+Um für verschiedene Plattformen und Betriebssysteme zur Verfügung zu stehen und um das Versprechen zu erfüllen, als einheitliche Schnittstelle zum Kassensystemen zu fungieren, ist die fiskaltrust.Middleware nach folgender Architektur aufgebaut.
 
```

Comment:
"Kassensystemen" --> "Kassen- oder Aufzeichnungssystem"

Reply:
Danke fürs Feedback, ich habe es in meinem aktuellen Commit berücksichtigt: https://github.com/christian-rogobete/productdescription-de-doc/commit/2805ee1b9398e0f0b6436d027e64f58da528cd86